### ISSUE TITLE
Bootstrap table classes not loading

### ISSUE THREAD

**Issue by tacman** [kevinpapst/AdminLTEBundle#135]
When I add a basic table to the demo home page, using standard bootstrap classes, the styling isn't applied.

```html
    <table class="table">
        <thead class="thead-tdark">
        <tr>
            <th scope="col">#</th>
            <th scope="col">First</th>
            <th scope="col">Last</th>
            <th scope="col">Handle</th>
        </tr>
        </thead>
    </table>
```

![image](https://user-images.githubusercontent.com/619585/81740977-cbcf6680-946b-11ea-9c62-9072a92f14f7.png)

When I try loading bootstrap separately, the table head comes out correctly, but everything else is smaller and all over the page.

**Comment by kevinpapst** [comment]
Bootstrap 4 classes in a bootstrap 3 environment doesn't work.

You need to use the v3 branch if you want bootstrap 4.

**Comment by tacman** [comment]
I'm using this:

        "kevinpapst/adminlte-bundle": "^3",

When I use the adminlte-3 version, the knp menus fail.  I spent an embarrassingly long time trying to fix that, and couldn't.  Any chance you could just get knp-menu with the adminlte-3 to work?  I think it's just the badge macro.  Thanks.

**Comment by kevinpapst** [comment]
You can test the latest commit in the v3 branches (both demo and this repo).
I fixed the kip problems, but please note that KNP is not yet fully functional, its still on the TODO list

**Comment by tacman** [comment]
Thanks, Kevin, appreciate that!

Indeed, the KNP nested menus aren't working, so I'll get at least one of my
sites converted over to using your menu event handler instead.

The demo app isn't working at the moment on the admin-lte3 branch.  (git
clone .., git checkout adminlte-3 , composer install, yarn install, yarn
run encore dev (or npm run build).  Something is missing, but I can't tell
what.

[image: image.png]

**Comment by kevinpapst** [comment]
the demo works fine on my end on the v3 branch, try from a fresh clone without running yarn

**Comment by tacman** [comment]
When you say "v3", do you mean adminlte-3 or 3.0?

**Comment by tacman** [comment]
Yeah, adminlte-3 does work right out of the box, but yarn run encore dev
shows this:

Running webpack ...

 ERROR  Failed to compile with 1 errors

               6:49:04 AM

This dependency was not found:

* admin-lte/dist/css/AdminLTE.min.css in
./vendor/kevinpapst/adminlte-bundle/Resources/assets/admin-lte.js

**Comment by kevinpapst** [comment]
Interesting, never used `yarn run encore dev` in the last year... if you fix it, please sent a PR

**Comment by tacman** [comment]
It seems like that's the most common practice for Symfony, no?  But npm run
build has the same problem.
